os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtk_coex.c

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2017, Realsil Semiconductor Corporation. All rights reserved.
+ *
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "trace_app.h"
+#include "vendor_cmd.h"
+#include "rtk_coex.h"
+#include "platform_stdlib.h"
+#include <bt_intf.h>
+
+struct rtl_btinfo
+{
+    uint8_t cmd;
+    uint8_t len;
+    uint8_t data[6];
+};
+
+static void rtk_notify_info_to_wifi(uint8_t length, uint8_t *report_info)
+{
+    uint8_t para_length = 1 + length;
+    uint8_t *p = (uint8_t *)malloc(para_length * sizeof(uint8_t));

```

Comment:
For loadable config, please use `kmm_malloc` instead of `malloc`.

Reply:
Already changed.